### PR TITLE
Add no var eslint check. Closes #27.

### LINKED ISSUE #27: Add 'no-var' rule to .eslintrc.json

### PR DESCRIPTION
**Pull request by chambrid**

### CODE REVIEW

File: src/lib/env.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ if (_.isString(HUBOT_IBMCLOUD_LDAP_READERUSERS_GROUP_DN_LIST)) {
 	HUBOT_IBMCLOUD_LDAP_READERUSERS_GROUP_DN_LIST = HUBOT_IBMCLOUD_LDAP_READERUSERS_GROUP_DN_LIST.trim();
 }
 
-const settings = {
+let settings = {
```
Comment (aeweidne) [inline src/lib/env.js]: why the change here?

Comment (chambrid) [reply src/lib/env.js]: fixed. was a find & replace issue.

**Comment by aeweidne** [comment]
+1
